.github/workflows/check.yml

```diff
@@ -38,11 +38,11 @@ jobs:
 
       - name: Run Unit tests.
         # skip integration testing
-        run: go test -p 1 -gcflags="all=-N -l" `go list ./... | grep -v github.com/machinefi/sprout/cmd/test` -covermode=atomic -coverprofile cover.out
+        run: go test -p 1 -gcflags="all=-N -l" `go list ./... | grep -v github.com/machinefi/sprout/cmd/e2etest` -covermode=atomic -coverprofile cover.out
```

Comment:
can del "-p 1"

Reply:
gomonkey is not thread safe, so can not del `-p 1`